### ISSUE TITLE
Too many labs beneath houses

### ISSUE THREAD

**Issue by Inglonias** [CleverRaven/Cataclysm-DDA#27484]
**Describe the bug**  
With the merging of #27462 the frequency of houses has gone up. This has, indirectly, led to increased frequency of labs
**To Reproduce**  
N/A

**Expected behavior**  
Cities shouldn't have four or five labs beneath them.

**Screenshots**  
![image](https://user-images.githubusercontent.com/2125926/50799685-09861300-12ab-11e9-8fac-1d1043f465db.png)
![image](https://user-images.githubusercontent.com/2125926/50799695-10ad2100-12ab-11e9-9f24-a57ff99c2135.png)

**Versions and configuration(please complete the following information):**
 - OS: Windows 10
 - Game Version: 0.C-36249-g7de8b69
 - Graphics version: Tiles
 - Mods loaded: N/A

**Additional context**  
Admittedly, this is a matter of opinion, but I'm not sure if this was something that was considered when houses were made more common.

**Comment by ZhilkinSerg** [comment]
You can try lowering value of `basement_hidden_lab_stairs`:

https://github.com/CleverRaven/Cataclysm-DDA/blob/4a1ab3493046cf682c929ea2933708c007a4fddd/data/json/regional_map_settings.json#L537

**Comment by sneeze-attack** [comment]
@Inglonias I've been doing some local testing with the value mentioned by ZhilkinSerg and when changing the value from `50` to `25` the results are much more reasonable.

I created a world with default settings (city size 8 and city spacing 4). I left the regional settings at default and checked 30 generated cities that contained at least 10 houses each. I found 33 basement labs in the 30 cities checked.

I created another world with the same settings but I changed regional settings for `basement_hidden_lab_stairs` to 25. I found 9 basement labs in the 30 cities checked.


Does that seem more acceptable?
